homework3/3_1.py

```diff
@@ -0,0 +1,27 @@
+"""ListTask."""
+
+
+list1 = ['a', 'b', 'c', 'd']
+list2 = [fir + la for fir in list1[0:2] for la in list1[1:]]
```

Comment:
In python for useless name we often use `foo` and `bar` and as useless string "the knights who say ni!"
:)